packages/themed-stylesheet/package.json

```diff
@@ -28,7 +28,7 @@
     "@types/react-native": "0.60.19",
     "@uifabricshared/build-native": "^0.1.1",
     "@uifabricshared/eslint-config-rules": "^0.1.1",
-    "react-native": "^0.59.0",
+    "react-native": "^0.60.0",
```

Comment:
Does this package use react-native?  I dont see it as either a dep or peerdep?   The answer to those questions would determine if we need to bump more things, and what kind of change type this change should be.

Reply:
If I remember right this package does use StyleSheet from react native.  I don't know that it needs the version bump, that API has been stable for some time.  Do all our peer dependencies in RN need to be aligned to make the build system happy or can each package specify its minimum required version?

Reply:
We probably want the dev deps aligned so that you dont have to download 100 versions of RN to work on the repo.  But the peerdeps should specify the full range of what the code would actually work with.   (Or what we support).  Which for now should probably be `>= 0.59 < 0.62` or something like that.

Reply:
@acoates-ms Yes it does. It has this code. 

`import { ViewStyle, TextStyle, ImageStyle, StyleSheet } from 'react-native';`

Reply:
Without this change, I'm getting build errors due to the fact that there're different versions of React-Native in the repo.

Reply:
@acoates-ms @JasonVMo This is a dev dep, so I think we should be good with this change?

Reply:
I tried reverting the change and the build still succeeded. So I'm reverting it now.